src/main/webapp/resources/js/components/lists/BasicList.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Component to render a basic list with just a
+ * title and description for each item in the list
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { List, Typography } from "antd";
+
+const { Text } = Typography;
+
+export function BasicList({
+  itemLayout = "horizontal",
+  dataSource = { dataSource },
+  ...props
+}) {
+  return (
+    <List
+      itemLayout="horizontal"
+      dataSource={dataSource}
+      renderItem={item => (
+        <List.Item>
+          <List.Item.Meta
+            title={<Text strong>{item.title}</Text>}
+            description={item.desc}
+          />
+        </List.Item>
+      )}
+    />
+  );
+}
+
+BasicList.propTypes = {
+  itemLayout: PropTypes.string,
+  dataSource: PropTypes.array.isRequired
```

Comment:
Can you look into `propTypes` for a list of objects that have a `title` and a `desc` attribute.

Reply:
Added in 48b1a7b8bb8e39cb2a7eda2d963f27f207aae066